cas-server-webapp/src/main/java/org/pac4j/http/IpProfile.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.pac4j.http;
+
+import org.pac4j.http.profile.HttpProfile;
+
+/**
+ * IP profile.
+ *
+ * @author Jerome Leleu
+ * @since 1.8.0
+ */
+public class IpProfile extends HttpProfile {
```

Comment:
Yes, definitely. And the `IpProfileCreator` also. I should have added a big warning: it's too early for a comprehensive review.

Handling practical use cases is a good way to check everything fits in the pac4j paradigm and add the missing components in pac4j.